omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -36,9 +35,18 @@
     <%      }
         }%>
     function handlePatientRowSelection() {
+    	var afterSelectedUrl = '${ ui.escapeJs(config.afterSelectedUrl) }';
         this.handle = function (row) {
             var uuid = row.uuid;
-            location.href = '/' + OPENMRS_CONTEXT_PATH + emr.applyContextModel('${ ui.escapeJs(config.afterSelectedUrl) }', { patientId: uuid, breadcrumbOverride: '${ ui.escapeJs(breadcrumbOverride) }'});
+            if(afterSelectedUrl && afterSelectedUrl != 'null') {
+            	location.href = '/' + OPENMRS_CONTEXT_PATH + emr.applyContextModel(afterSelectedUrl, { patientId: uuid, breadcrumbOverride: '${ ui.escapeJs(breadcrumbOverride) }'});
+        	} else {
+        		jQuery("#patient-search").attr("selected_uuid", uuid);
```

Comment:
Did you intentionally hard code the widget id?

Reply:
its id has always been patient-search, i just had to use the existing to set more attributes onto it